### DISCUSSION TITLE
Has anyone used stable diffusion for face obfuscation? So you look like yourself - but face recognition models don't recognize you. So you can safely use those images on social networks without the fear of being doxed. What are the best tools for that? Is Foocus UI a good choice for it?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [lllyasviel/Fooocus#1389] [General]
Has anyone used stable diffusion for face obfuscation? So you look like yourself - but face recognition models don't recognize you. So you can safely use those images on social networks without the fear of being doxed. What are the best tools for that? Is foocus UI a good choice?

Can you share your experience with it - what Loras did you use, prompts, and so on?

**Comment by jclevet** [comment]
Hi, How do you know that "face recognition models don't recognize you"?

**Reply by martinenkoEduard** [reply]
The goal is to modify your own face so you look like yourself, and can share this image on social networks, but you can't be found (doxed) by using face search algorithms. 

in practical terms - I will test it on my local face recognition software - something like ArcFace, AdaFace.
Maybe yousing compreface software.

**Comment by lllyasviel** [comment]
https://github.com/lllyasviel/Fooocus/discussions/414#detail
